nuget/bootstrap.nuspec

```diff
@@ -16,9 +16,9 @@
     <copyright>Copyright 2017</copyright>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <dependencies>
-      <dependency id="jQuery" version="3.0.0.1" />
+      <dependency id="jQuery" version="[1.9.1,3)" />
```

Comment:
Yes, but I did mess up values. Just updated to `[1.9.1,4)` which means greater or equal to v1.9.1 but less than v4